ext/opcache/Optimizer/zend_func_info.c

```diff
@@ -503,15 +441,12 @@ static const func_info_t func_infos[] = {
 	FN("array_key_last",               MAY_BE_NULL | MAY_BE_LONG | MAY_BE_STRING),
 	F1("pos",                          UNKNOWN_INFO),
 	F1("assert_options",               MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_LONG | MAY_BE_STRING | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_LONG | MAY_BE_ARRAY_OF_STRING | MAY_BE_ARRAY_OF_OBJECT | MAY_BE_OBJECT),
-	F0("version_compare",              MAY_BE_FALSE | MAY_BE_TRUE | MAY_BE_LONG),
 	F1("str_rot13",                    MAY_BE_STRING),
 	F1("stream_get_filters",           MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_LONG | MAY_BE_ARRAY_OF_STRING),
-	F0("stream_filter_register",       MAY_BE_FALSE | MAY_BE_TRUE),
 	F1("stream_bucket_make_writeable", MAY_BE_NULL | MAY_BE_OBJECT),
 	F1("stream_bucket_prepend",        MAY_BE_FALSE | MAY_BE_OBJECT),
 	F1("stream_bucket_append",         MAY_BE_FALSE | MAY_BE_OBJECT),
 	F1("stream_bucket_new",            MAY_BE_FALSE | MAY_BE_OBJECT),
-	F0("output_reset_rewrite_vars",    MAY_BE_FALSE),
```

Comment:
`MAY_BE_FALSE` on its own is not identical to what the stubs are for output_reset_rewrite_vars - User-defined functions aren't allowed to have `: false {` (MAY_BE_FALSE) on their own, and I assume internal functions would be consistent with that.

(The linked pr doesn't change output_reset_rewrite_vars)

```php
ext/standard/basic_functions.stub.php
42:function output_reset_rewrite_vars(): bool {}
```

Reply:
Thanks for the double check :)